src/main/java/edu/ucsb/cs156/gauchoride/repositories/ShiftRepository.java

```diff
@@ -0,0 +1,16 @@
+package edu.ucsb.cs156.gauchoride.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import edu.ucsb.cs156.gauchoride.entities.Shift;
+
+import java.util.Optional;
+
+@Repository
+public interface ShiftRepository extends CrudRepository<Shift, Long> {
+  Optional<Shift> findByDay(String day);
+  Optional<Shift> findByDriver(Long driverID);
+  // TODO: Find by shift would be nice to have, it might be a little difficult to implement
+  //Optional<Shift> findByShift(String shift);
```

Comment:
Remove unnecessary comments, alternatively you can create an issue on your kanban board

Reply:
Ok